### ISSUE TITLE
Custom Inserts do not work if config variables have been edited

### ISSUE THREAD

**Issue by bagelbyheart** [klembot/chapbook#129]
**Describe the bug**
[Custom Inserts](https://klembot.github.io/chapbook/guide/advanced/adding-custom-inserts.html) do not work outside of Test view if `config` options have been edited. This also breaks the feature going forward if cache is not cleared after correcting.

**To Reproduce**
1. Make a new chapbook 1.2.1 story on the [twine website](https://twinery.org/2/)
2. Add the `{smiley face}` example from the [chapbook manual](https://klembot.github.io/chapbook/guide/advanced/adding-custom-inserts.html)
3. Add the following to the only story passage  
```
config.header.left: '{passage.name}'
--
{smiley face}
```
4. Use the `Play` link to test the issue. Instead of a 😀 we just see `{smiley face}`

**Expected behavior**
Custom Inserts should work even when cosmetic config changes have been made, and moreso should behave consistently between `Test` and `Play` links from the Twine app.

**Stories and screenshots**
[testInsert.zip](https://github.com/klembot/chapbook/files/8874029/testInsert.zip)

**Environment**
So far this has been tested on Windows 10 in Firefox and Chrome. The story has been tested from the Twine webapp, as well as built from twee source using tweego.

**Comment by MoreFoxBeans** [comment]
I've been having the same issue, I'm just gonna port my story to SugarCube for the time being.

**Comment by HAL9000-GH** [comment]
I experienced this exact problem as well with the latest 1.2.3. I even had the problem persist beyond changing the code back. I had to clear the cache manually. A console error reveals that the code in the Story JavaScript doesn't see the `config` object properly, I believe.

It seems to only happen when the variables are changed at the top of the page (above the `--`) in a page that uses the `{inserts}`. It also affects `[modifers]`. However, I did have success with using JavaScript:
```
[javascript]
config.body.transition.name = "none";
config.header.center = "something";
[continue]
```
This seems to work without issue as a solid workaround. And it even applies it all before the passage renders. Meaning, even the transition applies immediately and not afterwards on the next passage transition.
